compiler/rustc_feature/src/active.rs

```diff
@@ -478,6 +478,10 @@ declare_features! (
     (active, object_safe_for_dispatch, "1.40.0", Some(43561), None),
     /// Allows using `#[optimize(X)]`.
     (active, optimize_attribute, "1.34.0", Some(54882), None),
+    /// Allows using `Option<&Option<T>>::flatten_ref()`,
+    /// Option<&mut Option<T>>::flatten_ref()` and
+    /// Option<&mut Option<T>>::flatten_mut()`
+    (active, option_reference_flattening, "CURRENT_RUSTC_VERSION", None, None),
```

Comment:
afaict the feature list here is only for features needing special compiler support, library-only features do not need to be added to this list.

Reply:
I was bamboozled by myself, thanks!